### PR TITLE
CITY-22: Add post "Co-design Session #3: Thresholds"

### PR DESCRIPTION
**Pull request by avtar**

### CODE REVIEW

File: content/ideas/co-design-session-3/index.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+## DAY 2
+
+### Design Activity: Part 3 - Prototype Challenge
+
+**Objective: Building a prototype to meet unrepresented needs<br>
+Duration: 1 hour 30 minutes**
+
+The second day began promptly with the prototyping challenge. Participants were given an hour and a half to choose an idea they brainstormed the previous day, expand on the idea, and build or prepare a demonstration of their idea. Each group was provided some prompting questions to help this process: Is it accessible? Is it safe? Does it support independence? What would you need? How can technology help? Who else is involved?
+
+{{< imgblock src="images/IMG_7863.jpg" resize="wide" alt="Pictured is group \"Group #2\" prototype. They have used lego blocks to build a platform, craft paper to build an elevator cube and plasticine to create a mobility device, such as an electric scooter made and a guide dog." >}}
```
Comment (avtar) [inline content/ideas/co-design-session-3/index.md]: Currently waiting to hear back from @sepidehshahi regarding this group's name. I left a comment about this in the Google doc used to create this post.

File: content/ideas/co-design-session-3/index.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+<ol>
+<li>In & Out: Moving between spaces
+<blockquote>
+"This theme explores thresholds that we encounter moving in and out of spaces, for example moving in and out of buildings, private or public spaces or places with open or restricted access. We want you to think of your personal experiences transitioning between spaces. How do you think these thresholds limit or enable your urban experience?"
+</blockquote></li>
+
+<li>Up & Down: Moving between different levels of a space 
+<blockquote>
+"This theme explores thresholds that exist between different levels of a space for example going from one level of a building to another. With this theme we want you to think of the different types of thresholds that are encountered moving up and down, and the challenges that exist in crossing these thresholds? How can these transitions be made more seamless?"
```
Comment (michelled) [inline content/ideas/co-design-session-3/index.md]: The second sentence should not have a question mark.

**Comment by michelled** [comment]
We decided that for text heavy photos we would provide transcripts. See this post for an example: https://cities.inclusivedesign.ca/ideas/co-design-session-2/

File: content/ideas/co-design-session-3/index.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+{{< imgblock src="images/IMG_1862.jpg" resize="wide" alt="The image above shows examples of items used for the warm up activity printed on braille cards. The cards pictured have names of items such as \"bag of beans\", \"cupcakes\", \"pressured air\", \"marshmallows\", \"water\" and \"keyboards\"" >}}
```
Comment (michelled) [inline content/ideas/co-design-session-3/index.md]: The alt text probably shouldn't read 'The image above'.

File: content/ideas/co-design-session-3/index.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+* Challenge: Imagine a better experience in the future for moving between different levels
+
+Getting Around:
+
+* Challenge: Imagine a better experience in the future for switching modes of transportation
+
+As the groups engaged in discussion, the assigned facilitators provided insight and guided the discussion to reflect on each theme. Each group then had the opportunity to briefly share their ideas with the larger group and received feedback before prototyping their idea the next day. The end of the group discussion on preliminary ideas marked the end of the first half of this two-day session. Participants were encouraged to reflect on their ideas and be more observant of the thresholds over the following day. 
+
+{{< imgblock src="images/IMG_7848.jpg" resize="wide" alt="Pictured above are the ideas generated by different groups around personal encounters with thresholds in the city, and reflections of positive and negative experiences associated with these encounters. The responses listed are from groups \"Hedgehog\", \"The A team\",\"Challenging\", \"Cats and Dogs\", \"Group#2\" and \"Assorted Barrier Busters\"" >}}
```
Comment (michelled) [inline content/ideas/co-design-session-3/index.md]: This alt text says 'Pictured above' we might as well change that too.

File: content/image-descriptions/co-design-session-3/group-discussion-whiteboard.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+* Taxis {{< arrow >}} hard to flag 
+  * Cant see meter, only uber has an app
+* More space to sit on sidewalk 
+  * Not in way to prevent tripping
+
+## Cats and Dogs group
+
+* Beacon technology {{< arrow >}} doors that slide based on beacon
+* Building surrounds no snow and ice
+* Tactile surfaces leading to elevator {{< arrow >}} ore receptionist to help
```
Comment (michelled) [inline content/image-descriptions/co-design-session-3/group-discussion-whiteboard.md]: Typo here 'ore'

File: content/image-descriptions/co-design-session-3/group-discussion-whiteboard.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+* Textured edges {{< arrow >}} railings/ramps
+* Sensitivity training {{< arrow >}} know what to say
+* Equal access opportunities {{< arrow >}} elevators near stairs
+* System of monitoring how things are working
+* Contrast strips {{< arrow >}} illuminate edges/textured gradients
+* Curb cuts in center marked with colour/texture to get out of vehicle safely
+
+## Assorted Barrier Busters group
+
+* Cab pulling up to office building in January {{< arrow >}} ram and road clear of debris {{< arrow >}} some vehicle that docks at cab for smooth transition/transfer which carries you into building {{< arrow >}} recall your accessibility preferences based on badge {{< arrow >}} app to be a local helper.
```
Comment (michelled) [inline content/image-descriptions/co-design-session-3/group-discussion-whiteboard.md]: I think there's a typo here 'ram' instead of 'ramp'

**Pull request merged**